app/controllers/articles/wips_controller.rb

```diff
@@ -4,15 +4,15 @@ class Articles::WipsController < ApplicationController
   before_action :require_admin_or_mentor_login, only: %i[index]
 
   def index
-    @articles = list_articles
+    @articles = wip_articles
     @articles = @articles.tagged_with(params[:tag]) if params[:tag]
     render layout: 'welcome'
   end
 
   private
 
-  def list_articles
-    articles = Article.with_attached_thumbnail.includes(user: { avatar_attachment: :blob }).order(created_at: :desc).page(params[:page])
-    articles.where(wip: true)
+  def wip_articles
+    articles = Article.with_attached_thumbnail.includes(user: { avatar_attachment: :blob })
+    articles.where(wip: true).order(created_at: :desc).page(params[:page])
```

Comment:
いったん変数に入れる必要は無いように思いました。

```suggestion
    Article.with_attached_thumbnail.includes(user: { avatar_attachment: :blob })
      .where(wip: true).order(created_at: :desc).page(params[:page])
```